tensorrt_llm/_torch/auto_deploy/transform/library/kvcache_transformers.py

```
@@ -0,0 +1,267 @@
+        attention_mask: Optional[torch.Tensor],
+        **kwargs,
+    ):
+        # check if we need to transpose the inputs
+        # incoming layout is bnsd in HF attn interface
+        attention_layout = attn_descriptor.get_attention_layout()
+        if attention_layout == "bsnd":
+            query = query.transpose(1, 2)
+            key = key.transpose(1, 2)
+            value = value.transpose(1, 2)
+        elif attention_layout != "bnsd":
+            raise ValueError(f"Unsupported attention layout: {attention_layout}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)